powerauth-push-server/src/main/java/com/wultra/push/service/PushSendingWorker.java

```diff
@@ -152,15 +153,15 @@ void sendMessageToFcm(final FcmClient fcmClient, final PushMessageBody pushMessa
             final FcmSuccessResponse response = responseEntity.getBody();
             if (response != null && response.getName() != null) {
                 if (response.getName().matches(FCM_RESPONSE_VALID_REGEXP)) {
-                    logger.info("Notification sent successfully, response: {}.", response.getName());
+                    logger.info("action: sendMessageToFcm, state: succeeded, response: {}", response.getName());
                     callback.didFinishSendingMessage(PushSendingCallback.Result.OK);
                 } else {
-                    logger.error("Invalid response received from FCM, notification sending failed - unexpected response name: {}.", response.getName());
+                    logger.warn("action: sendMessageToFcm, state: failed, response: {}", response.getName());
                     callback.didFinishSendingMessage(PushSendingCallback.Result.FAILED);
                 }
             } else {
                 // This state should not happen, only in case when response from server is invalid
-                logger.error("Invalid response received from FCM, notification sending failed - empty or invalid response.");
+                logger.warn("action: sendMessageToFcm, state: failed, error: {}, response: {} ", "empty or invalid response", response);
```

Comment:
This seems overengineering to me.
```suggestion
                logger.warn("action: sendMessageToFcm, state: failed, error: empty or invalid response, response: {} ", response);
```